xbmc/windowing/egl/EGLWrapper.cpp

```diff
@@ -22,6 +22,8 @@
 #ifdef HAS_EGL
 #include "utils/log.h"
 #include "EGLNativeTypeAndroid.h"
+#include "EGLNativeTypeAmlAndroid.h"
+#include "EGLNativeTypeRKAndroid.h"
```

Comment:
sorry, I haven't noticed the ping.
I think we should not continue this very weird style of including everything and ifdeffing each implementation of the methods in the body. I see no reason why we should compile and include RPi specific files in Android build and vice versa.